src/inspector/main_thread_interface.h

```diff
@@ -94,6 +95,7 @@ class MainThreadInterface :
   // when we reenter the DispatchMessages function.
   MessageQueue dispatching_message_queue_;
   bool dispatching_messages_ = false;
+  bool stop_waiting_for_frontend_event_requested_ = false;
```

Comment:
Can you add a comment to explain the usage and reason for this flag?

Reply:
Thanks for the review. Applied. PTAL.